src/Text/Layout/Table.hs

```diff
@@ -211,15 +229,15 @@ gridBWithCMIs specs tab = (zipWith4 columnModifier positions cms cMIs <$> tab, c
     positions = map position specs
     cms = map cutMark specs
 
--- | A version of 'gridB' specialised to produce 'String's.
```

Comment:
`gridLines` and `gridString` both say “to produce `String`s”. We should probably be consistent about which phrasing we use.

Reply:
Yes, consistency is nice.